### PR TITLE
Minibomb Buff (+ slightly better explosive descriptions)

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Revival of #13715
Lowers the timer of the minibomb from 10 seconds to 5, like it is in SS13.
Makes the minibomb timer sound less boring (and quieter.)
Also changed some explosive descriptions & uplink descriptions to be less bad while I was there. A lot of parts of them were taken from /tg/station wholesale.

This PR used to heavily buff the explosion radius of the minibomb, but after some testing I found out that it was already mostly similar to how it was in SS13.
This probably won't fix explosions being shitty in traitor rounds as a whole, but reduced timer should make it much more useful, allowing it to actually do damage before people are able to react most of the time (as opposed to people running out of the room and then immediately fixing the hole before it can do any damage.)

**Media**
This video is hilariously botched and does not show the actual explosion, but should at least show you how the new timer sounds:

[Content.Client_5i0AZw9iTw.webm](https://github.com/space-wizards/space-station-14/assets/78941145/b1bd7582-4d38-4643-9961-6ff66e2633f6)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: The Syndicate has retrofitted the minibomb with a higher quality timer. The fuse has been halved to five seconds, and the beeping sound is now much quieter (and cooler.)
- tweak: Changed some descriptions of explosives both in-and-out of the uplink to be less weird.
- tweak: Fixed the EMP grenade's capitalization.

### CODE REVIEW

**Comment by liltenhead** [comment]
I like it

**Comment by Myakot** [comment]
I like the radius, however the issues you're describing at the start of your text are... not really there? Minibomb gibbing is for quick kills, or even scummier - stam crits, dropping an activated nade and running out. It's a localized devastation of unarmored targets.

tl;dr, I agree with you, but current minibomb is not nearly as shit as you describe it to be.

**Comment by asperger-sind** [comment]
Going to rebalance [#17292] a bit if this gets merged

**Comment by Flareguy** [comment]
> I like the radius, however the issues you're describing at the start of your text are... not really there? Minibomb gibbing is for quick kills, or even scummier - stam crits, dropping an activated nade and running out. It's a localized devastation of unarmored targets.
> 
> tl;dr, I agree with you, but current minibomb is not nearly as shit as you describe it to be.

Fair enough, but you have no idea how many times I've seen someone run into an area, toss the minibomb expecting it to fuck things up (read: deal actually meaningful hull damage,) and then have the hole get patched in mere seconds. It is probably one of the worst feelings you can have as an observer in this game.

File: Resources/Locale/en-US/store/uplink-catalog.ftl [hunk 1]
```diff
@@ -28,13 +28,13 @@ uplink-gloves-north-star-desc = A pair of gloves that reduce your punching coold
...
-uplink-explosive-grenade-desc = Grenade that creates a small but devastating explosion.
+uplink-explosive-grenade-desc = A simplistic grenade with a ten-second fuse that is geared towards injuring personnel. Causes minimal hull damage.
 
 uplink-flash-grenade-name = Flashbang
-uplink-flash-grenade-desc = Eeeeeeeeeeeeeeeeeeeeee
+uplink-flash-grenade-desc = A standard-issue flashbang, capable of blinding and slowing down anyone without proper protection. This, of course, includes you; make sure you're properly equipped before using it.
 
 uplink-mini-bomb-name = Minibomb
-uplink-mini-bomb-desc = A precision sabotage explosive for quickly destroying a machine, dead body, or whatever else needs to go.
+uplink-mini-bomb-desc = A messy medium-yield explosive with a ten-second long fuse. Perfect for causing chaos, destroying small rooms, or finishing-up unfinished business.
```
Comment (keronshb) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:37]:
```suggestion
uplink-mini-bomb-desc = A messy medium-yield explosive with a ten-second long fuse. Perfect for causing chaos, destroying small rooms, or finishing-up unfinished business.
```

Move minibomb down to 3-4 seconds for a fuse.

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 2]
```diff
@@ -101,9 +100,9 @@
   - type: ExplodeOnTrigger
   - type: Explosive
     explosionType: Minibomb
-    totalIntensity: 200
-    intensitySlope: 30 #Will destroy the tile under it reliably, space 1-2 more to rods. Only does any significant damage in a 5-tile cross.
-    maxIntensity: 60
+    totalIntensity: 700
+    intensitySlope: 16 #If it's any lower then it feels really weak. If you're nerfing the minibomb, probably don't change just this value!
+    maxIntensity: 295
```
Comment (keronshb) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:105]: This is also going to effect the minibomb implant. Please revert it back and lower the timer to 3-4 seconds.
If you want a larger bomb with a long timer, add the macrobomb (with bomb diffusal) instead.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 3]
```diff
@@ -124,7 +124,7 @@
   description: uplink-mini-bomb-desc
   productEntity: SyndieMiniBomb
   cost:
-    Telecrystal: 6
+    Telecrystal: 8
```
Comment (keronshb) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:127]:
```suggestion
    Telecrystal: 6
```

With the timer lowered and the original explosion size, should be back down to 6TC.

File: Resources/Locale/en-US/store/uplink-catalog.ftl [hunk 4]
```diff
@@ -28,13 +28,13 @@ uplink-gloves-north-star-desc = A pair of gloves that reduce your punching coold
 
 # Explosives
 uplink-explosive-grenade-name = Explosive Grenade
-uplink-explosive-grenade-desc = Grenade that creates a small but devastating explosion.
+uplink-explosive-grenade-desc = A simplistic grenade with a ten-second fuse that is geared towards injuring personnel. Causes minimal hull damage.
 
 uplink-flash-grenade-name = Flashbang
-uplink-flash-grenade-desc = Eeeeeeeeeeeeeeeeeeeeee
+uplink-flash-grenade-desc = A standard-issue flashbang, capable of blinding and slowing down anyone without proper protection. This, of course, includes you; make sure you're properly equipped before using it.
```
Comment (keronshb) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:34]:
```suggestion
uplink-flash-grenade-desc = Eeeeeeeeeeeeeeeeeeeeee
```

You don't have to remove the memey explanation. Some of it is part of the spirit of the game.

**Review by keronshb — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml (hunk 2 above)
Comment (Flareguy) [reply Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:105]: Hm... I'm not sure what to think. The macrobomb explosion is _gigantic_ - nearly on the level of the syndicate bomb from SS13. I would be fine with making another bomb prototype as to not change the minibomb implant (and probably renaming it to something like the "explosive implant,") but I feel like this kind of defeats the purpose of the PR.
This would just make it better at gibbing stuff, which would probably be best suited to something like X-4 if we're going with SS13 analouges.

Additionally, a macrobomb would entail either nerfing the hell out of the implant or just making the syndicate bomb (which is a different item in purpose and function entirely, and also not something I do as I can't code.)

File: Resources/Locale/en-US/store/uplink-catalog.ftl (hunk 4 above)
Comment (Flareguy) [reply Resources/Locale/en-US/store/uplink-catalog.ftl:34]: It's just a copy of the item description, like most other uplink items. The comedy is already in there so I'm not sure if it's worth having both as it just ends up being weirdly non-descriptive all-around.

Comment (keronshb) [reply Resources/Locale/en-US/store/uplink-catalog.ftl:34]: Should be fine then.

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml (hunk 2 above)
Comment (keronshb) [reply Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:105]: I meant making a separate syndie bomb/macrobomb item, not to replace the minibomb with it.

**Comment by Vaaankas** [comment]
its a MINIbomb, i dont think its reasonable to make the explosion that strong.
Instead i would suggest you introduce an alternative, larger hull-breaching bomb, like in #15380, that way we would have both an easy-gib version, and a "goodbye atmo" version.

**Review by keronshb — APPROVED** [review]
